src/NumericalAlgorithms/DiscontinuousGalerkin/Actions/ApplyFluxes.hpp

```diff
@@ -31,57 +31,69 @@ struct Mesh;
-/// \ingroup ActionsGroup
-/// \ingroup DiscontinuousGalerkinGroup
-/// \brief Compute boundary contribution to the time derivative for
-/// use in global time stepping.
-///
-/// Uses:
-/// - ConstGlobalCache: Metavariables::normal_dot_numerical_flux
-/// - DataBox:
-///   - Tags::Mesh<volume_dim>
-///   - Tags::Mortars<Tags::Mesh<volume_dim - 1>, volume_dim>
-///   - Tags::Mortars<Tags::MortarSize<volume_dim - 1>, volume_dim>
-///
-/// DataBox changes:
-/// - Adds: nothing
-/// - Removes: nothing
-/// - Modifies:
-///   db::add_tag_prefix<Tags::dt, variables_tag>,
-///   FluxCommunicationTypes<Metavariables>::mortar_data_tag
-struct ApplyBoundaryFluxesGlobalTimeStepping {
-  template <typename DbTags, typename... InboxTags, typename Metavariables,
-            typename ArrayIndex, typename ActionList,
-            typename ParallelComponent>
+/*!
+ * \ingroup ActionsGroup
+ * \ingroup DiscontinuousGalerkinGroup
+ * \brief Compute element boundary contributions to the temporal step of the
+ * variables
+ *
+ * This action invokes the numerical flux operator for each mortar of the
+ * element, lifts the data to the volume and adds it to the temporal step
+ * variables. These would be the time derivatives `Tags::dt` of the system
+ * variables for an evolution system, for instance.
+ *
+ * With:
+ * - `flux_comm_types = dg::FluxCommunicationTypes<Metavariables>`
+ *
+ * Uses:
+ * - All items used by `flux_comm_types`
+ * - Metavariables:
+ *   - `temporal_id::step_prefix`
+ * - System:
+ *   - `volume_dim`
+ *   - `variables_tag`
+ * - ConstGlobalCache:
+ *   - `Metavariables::normal_dot_numerical_flux`
+ * - DataBox:
+ *   - `Tags::Mesh<volume_dim>`
+ *   - `Tags::Mortars<Tags::Mesh<volume_dim - 1>, volume_dim>`
+ *   - `Tags::Mortars<Tags::MortarSize<volume_dim - 1>, volume_dim>`
+ *
+ * DataBox changes:
+ * - Modifies:
+ *   - `db::add_tag_prefix<step_prefix, variables_tag>`
+ *   - `flux_comm_types::mortar_data_tag`
+ */
+template <typename Metavariables>
+struct ApplyFluxes {
+  template <typename Tag>
+  using step_prefix =
+      typename Metavariables::temporal_id::template step_prefix<Tag>;
+
+  template <typename DbTags, typename... InboxTags, typename ArrayIndex,
+            typename ActionList, typename ParallelComponent>
   static auto apply(db::DataBox<DbTags>& box,
                     const tuples::TaggedTuple<InboxTags...>& /*inboxes*/,
                     const Parallel::ConstGlobalCache<Metavariables>& cache,
                     const ArrayIndex& /*array_index*/,
                     const ActionList /*meta*/,
                     const ParallelComponent* const /*meta*/) noexcept {
-    static_assert(not Metavariables::local_time_stepping,
-                  "ApplyBoundaryFluxesGlobalTimeStepping cannot be used with "
-                  "local time-stepping.");
-
     using system = typename Metavariables::system;
-    constexpr size_t volume_dim = system::volume_dim;
+    constexpr size_t Dim = system::volume_dim;
```

Comment:
we use `snake_case` for variable names, `dim` would be preferred :)

Reply:
agreed